assetsTransformer.js

```diff
@@ -0,0 +1,7 @@
+const path = require("path");
+
+module.exports = {
+  process(src, filename, config, options) {
+    return "module.exports = " + JSON.stringify(path.basename(filename)) + ";";
```

Comment:
I ran into this [Invalid or unexpected token](https://stackoverflow.com/questions/54627028/jest-unexpected-token-when-importing-css) issue when I tried to execute the test.  I added this setting in the package.json to avoid transforming CSS import.
```
"jest": {
    "moduleNameMapper": {
      "\\.(css|less|scss)$": "<rootDir>/assetsTransformer.js"
    },
```
I updated the file to only export an empty object.
`module.exports = {};`